fdbserver/RestoreRoleCommon.actor.h

```diff
@@ -52,7 +52,8 @@ struct RestoreMasterData;
 
 struct RestoreSimpleRequest;
 
-typedef std::map<Version, MutationsVec> VersionedMutationsMap;
+//typedef std::map<Version, MutationsVec> VersionedMutationsMap;
```

Comment:
delete

Reply:
hmm, I must have forgotten to push my changes. This line was not on my local repo.